cmd/eval.go

```diff
@@ -19,10 +19,12 @@ import (
 var (
 	ErrEmptyFile      = errors.New("empty file")
 	FilePathArgsIndex = 0
+	useSolvedAhead    bool
 )
 
 func init() {
 	evalCmd.Flags().StringToStringP("replacements", "r", map[string]string{}, "Key value pair of comma delimited values. Example: 'NAMESPACE=foo,bar'")
+	evalCmd.Flags().BoolVarP(&useSolvedAhead, "presolve", "p", false, "Toggles how combinations are generated. When applied combinations are generated all at once compared to iteratively.")
```

Comment:
Let's avoid using initializing a global variable here, and instead get the value of the "presolve" option in the [`RunE(...)`](https://github.com/operator-framework/combo/blob/74719d2173a762660e7745b00f221bb2f51c52e8/cmd/eval.go#L73) function. You can find an example of this on line 74 where we get the value of the `replacements` CLI flag.

```go
useSolveAhead, err := cmd.Flags().GetBool("presolve")
if err != nil {
  return err
}
...
```

Reply:
Got it. I implemented the change, and it will be pushed in the next commit. Waiting to make commit until we can get some feed back on what Tyler thinks about changing `WithSolveAhead` to take a bool.